src/browser/index.js

```diff
@@ -42,17 +43,40 @@ if (isReload) {
 document.body.classList.toggle('use-native-titlebar',
                                Runtime.useNativeTitlebar());
 
+const replay =
+  () => {
+    const request = new XMLHttpRequest();
+    // We use sync XHR as we have nothing else to render otherwise.
```

Comment:
I'd prefer. It's just about getting the same behavior with gecko and servo. But if it's too much work, just forget about it.